app/views/main.scala.html

```diff
@@ -59,17 +59,23 @@
         @content
 
     </div>
-
-    @if(url.get == "/audit") {
-
-        <script type="text/javascript">
-            document.getElementsByTagName("body")[0].style.overflow = "hidden";
-            document.getElementById("wrap").style.padding = "0 0 35px";
-        </script>
+    @if(url.get != "/mobile") {
+        <head>
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+        </head>
+    }
+    @if(url.get == "/audit" || url.get == "/signInMobile" || url.get == "/signUpMobile") {
+
+        @if(url.get == "/audit") {
```

Comment:
I think it's kind of confusing to have this inside that outer if block. I realize that it is technically slightly less efficient to do this check outside of the outer if statement, but I think it reads more clearly that way.